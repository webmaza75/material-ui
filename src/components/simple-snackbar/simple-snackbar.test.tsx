import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import SimpleSnackbar from './simple-snackbar';

describe(`SimpleSnackbar`, () => {
  it(`SimpleSnackbar renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const result = renderer
      .render(<SimpleSnackbar
        isOpenUndoDeleteSnackbar={false}
        onItemsExactlyDelete={jest.fn()}
        onItemsUndoDelete={jest.fn()}
      />);

    expect(result).toMatchSnapshot();
  });
});
