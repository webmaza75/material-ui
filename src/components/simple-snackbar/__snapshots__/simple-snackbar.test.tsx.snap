// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleSnackbar SimpleSnackbar renders correctly 1`] = `
<ForwardRef(WithStyles)
  ContentProps={
    Object {
      "aria-describedby": "message-id",
    }
  }
  action={
    Array [
      <ForwardRef(WithStyles)
        color="secondary"
        id="undoDelete"
        onClick={[MockFunction]}
        size="small"
      >
        UNDO
      </ForwardRef(WithStyles)>,
      <ForwardRef(WithStyles)
        aria-label="close"
        className="makeStyles-close-1"
        color="inherit"
        id="closeSnackbar"
        onClick={[MockFunction]}
      >
        <Memo />
      </ForwardRef(WithStyles)>,
    ]
  }
  anchorOrigin={
    Object {
      "horizontal": "center",
      "vertical": "bottom",
    }
  }
  autoHideDuration={6000}
  id="snackBarForUndoDelete"
  message={
    <span
      id="message-id"
    >
      Task deleted
    </span>
  }
  open={false}
/>
`;
